guild.go

```diff
@@ -1191,6 +1190,21 @@ func (g guildQueryBuilder) GetRoles() ([]*Role, error) {
 	return getRoles(r.Execute)
 }
 
+// Search through GetRoles to find a role
+func (g guildRoleQueryBuilder) Get() (*Role, error) {
+	guild := g.client.Guild(g.gid)
+	roles, err := guild.GetRoles()
+	if err != nil {
+		return &Role{}, err
```

Comment:
return `nil, err` instead, no need to instantiate a empty role here. Same for the bottom statement

Reply:
done